src/tests/system/tests/test_failover.py

```diff
@@ -80,3 +80,30 @@ def test_failover__reactivation_timeout_is_honored(
     assert (
         f"Primary server reactivation timeout set to {expected} seconds" in log
     ), f"'Primary server reactivation timeout set to {expected} seconds' not found in logs!"
+
+
+@pytest.mark.importance("low")
```

Comment:
Can we make this a generic provider test? 

or 

Move this test into test_ipa.py ?